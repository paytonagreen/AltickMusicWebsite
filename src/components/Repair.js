import React from 'react';
import Placeholder from './Placeholder';

const Repair = () => {
  return (
    <div>
      <Placeholder/>
    </div>
  )
};

export default Repair;