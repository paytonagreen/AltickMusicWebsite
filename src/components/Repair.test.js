import React from 'react';
import { shallow } from 'enzyme';
import Repair from './Repair';

describe('Repair', () => {
  const repair = shallow(<Repair/>);

  it('renders correctly', () => {
    expect(repair).toMatchSnapshot();
  })
})