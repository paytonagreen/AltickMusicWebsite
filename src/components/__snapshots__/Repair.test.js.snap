// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repair renders correctly 1`] = `
<div>
  <Placeholder />
</div>
`;
